### DISCUSSION TITLE
(nodejs16) Migrate package

### DISCUSSION THREAD

**Discussion by tomchiverton** [chocolatey-community/chocolatey-packages#2060] [Migrations]
I was asked by the community reviewers of my own package of Nodejs v16 to see if you would consider shipping it - essentially just a meta package that depends on the latest v16 release.

There's clearly a need for a package that pins a particular supported upstream major version, rather than tracking the latest and greatest LTS.

Their view is that ss the Chocolatey Community Chocolatey Packages repository maintains the nodejs package, I think it would make sense for this package to be owned there too. 

Would you be interested in taking it on ? Attaching the meta package file here for discussion on it if required.

If not, hopefully the review team would except it independently.

[nodejs16.nuspec.txt](https://github.com/chocolatey-community/chocolatey-packages/files/9979232/nodejs16.nuspec.txt)

**Comment by pauby** [comment]
I think it makes sense to bring this in here. @AdmiringWorm what are your thoughts?

**Reply by AdmiringWorm** [reply]
As we own the original nodejs package it makes sense to add this package to this repository as well.

I can create an issue to target for this work. @tomchiverton would you be interested in creating a PR to add the package?

EDIT: I have created issue #2090 for this, if you @tomchiverton could comment on the issue (if you will be creating a PR), then we can assign the issue to you.
